src/configuration.ts

```diff
@@ -96,8 +96,8 @@ export class Configuration {
           config.section,
           config.name,
           config.value,
-          config.scope
-        )
+          config.scope,
```

Comment:
We're using this config:

https://github.com/Shopify/web-configs/blob/main/packages/prettier-config/index.json

which has `"trailingComma": "all"` meaning it would match the new default in Prettier 3, but it seems it wasn't being enforced previously... perhaps because `eslint-plugin-prettier` wasn't in use?